### PR TITLE
feat: add grafana dashboarding

### LINKED ISSUE #40: Add grafana monitoring with default config
Use latest grafana as loads-broker now does, configured with a sane default.

### PR DESCRIPTION
**Pull request by bbangert**
Grafana is now setup, configured with appropriate datasources and
populated with a dashboard from a S3 JSON file.

Closes #40

### CODE REVIEW

File: ardere/step_functions.py [hunk 1]
```diff
@@ -127,6 +135,58 @@ def _validate_plan(self):
...
+            os.environ["metrics_bucket"],
+            s3_filename
+        )
+        file_contents = dash_file.get()['Body'].read().decode('utf-8')
+        dash_contents = json.loads(file_contents)
+        dash_contents["title"] = dashboard_options["name"]
+        dash_contents["id"] = None
+
+        response = requests.post(grafana_url+"/api/dashboards/db",
+                                 auth=('admin', 'admin'),
```
Comment (jrconlin) [inline ardere/step_functions.py]: Nope, this goes into a config or env or anything else. 

Also, probably want to change this password.

File: ardere/step_functions.py [hunk 2]
```diff
@@ -127,6 +135,58 @@ def _validate_plan(self):
...
+        return "db/{}".format(response.json()["slug"])
+
+    def _ensure_dashboard(self, grafana_url, dashboard_options):
+        # type: (str, Dict[str, Any]) -> str
+        """Ensure the dashboard is present"""
+        dash_name = dashboard_options["name"]
+
+        # Verify whether the dashboard exists
+        response = requests.get(grafana_url+"/api/search",
+                                auth=('admin', 'admin'),
```
Comment (jrconlin) [inline ardere/step_functions.py]: same as last, nope.

File: ardere/step_functions.py [hunk 3]
```diff
@@ -187,7 +247,29 @@ def ensure_metrics_available(self):
         # Create an influxdb for this run
         influx_client = InfluxDBClient(host=metric_ip)
         influx_client.create_database(self.ecs.influx_db_name)
+
+        # Setup the grafana datasource
+        grafana_url = "http://{}:3000".format(metric_ip)
+        ds_api_url = "{}/api/datasources".format(grafana_url)
+        requests.post(ds_api_url, auth=('admin', 'admin'), json=dict(
```
Comment (jrconlin) [inline ardere/step_functions.py]: and again, nope.

File: ardere/aws.py [hunk 4]
```diff
@@ -122,6 +114,10 @@ class ECSManager(object):
 
     influxdb_container = "influxdb:1.1-alpine"
```
Comment (pjenvey) [inline ardere/aws.py]: you can upgrade to influxdb 1.2-alpine whenever, it's fully backwards compat

File: ardere/aws.py [hunk 5]
```diff
@@ -457,8 +519,8 @@ def create_services(self, steps):
         # type: (List[Dict[str, Any]]) -> None
         """Create ECS Services given a list of steps"""
         with ThreadPoolExecutor(max_workers=8) as executer:
-            results = executer.map(self.create_service, steps)
-        return list(results)
+            executer.map(self.create_service, steps)
+        return None
```
Comment (pjenvey) [inline ardere/aws.py]: might as well kill this line

**Pull request merged**